### PR TITLE
Zoom fixes

### PR DESCRIPTION
**Pull request by alejandrocoria**

### CODE REVIEW

File: common/src/main/java/journeymap/api/v2/client/display/Overlay.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ public Overlay setLabel(@Nullable String label)
     }
 
     /**
-     * The minimum zoom level (0 is lowest) where the polygon should be visible.
+     * The minimum zoom level (UIState.FULLSCREEN_ZOOM_MIN is lowest) where the overlay should be visible.
```
Comment (alejandrocoria) [inline common/src/main/java/journeymap/api/v2/client/display/Overlay.java:156]: Is it okay to mention the constant or should I mention the value directly?

File: common/src/main/java/journeymap/api/v2/client/display/Overlay.java [hunk 2]
```diff
@@ -297,8 +297,18 @@ public boolean isActiveIn(UIState uiState)
...
+
+    /**
+     * Whether the zoom of the given context is within the zoom range of the overlay.
+     *
+     * @param uiState    UIState
+     * @return true if the zoom of uiState is within the zoom range of the overlay
+     */
+    public boolean isInZoomRange(UIState uiState)
+    {
+        return (this.minZoom <= uiState.zoom && this.maxZoom >= uiState.zoom);
```
Comment (alejandrocoria) [inline common/src/main/java/journeymap/api/v2/client/display/Overlay.java:311]: I separated the zoom check so I could fix a bug later in Journeymap.

File: common/src/main/java/journeymap/api/v2/client/display/Overlay.java (hunk 1 above)
Comment (mysticdrew) [reply common/src/main/java/journeymap/api/v2/client/display/Overlay.java:156]: that is fine

File: common/src/main/java/journeymap/api/v2/client/display/Overlay.java (hunk 2 above)
Comment (mysticdrew) [reply common/src/main/java/journeymap/api/v2/client/display/Overlay.java:311]: nice

**Pull request merged**